src/components/EnvironmentOverride/DeploymentTemplateOverrideForm.tsx

```diff
@@ -532,7 +623,9 @@ export default function DeploymentTemplateOverrideForm({
             className={`deployment-template-override-form h-100 ${state.openComparison ? 'comparison-view' : ''} ${
                 state.showReadme ? 'readme-view' : ''
             }`}
-            onSubmit={handleSubmit}
+            onSubmit={(e) => {
```

Comment:
it can be written as `onSubmit={handleSaveChanges}`

Reply:
done